server/events.py

```diff
@@ -78,11 +78,13 @@ class SyncData(EventData):
         code: The code that already exists in the room.
         collaborators: The list of users that already collaborate in the room.
         time: The elapsed time since the creation of the room.
+        owner: The id of the owner of the room.
```

Comment:
Why make it optional? This data could be sent with all sync responses and requests, no?